gapic-generator-cloud/templates/cloud/gemspec.erb

```diff
@@ -1,6 +1,7 @@
 <%- assert_locals api -%>
 # -*- ruby -*-
 # encoding: utf-8
+require File.expand_path("../lib/<%= api.gem_version_require %>", __FILE__)
 
 Gem::Specification.new do |gem|
   gem.name          = "<%= api.gem_name %>"
```

Comment:
`gem.version` below this line need to be updated to use the version file.

Reply:
Great catch. Fixed.